tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "target": "esnext",
     "lib": ["esnext"],
     "strict": true,
-    "moduleResolution": "node",
+    "moduleResolution": "node16",
```

Comment:
Why can't we use `nodenext`? `node16` seems fairly old at this point (admittedly I only have a hazy understanding of what this config option does, so this may not be a real concern)